packages/flutter_tools/lib/executable.dart

```diff
@@ -138,17 +137,14 @@ Future<int> run(List<String> args, List<FlutterCommand> subCommands, {
     // Initialize the system locale.
     await intl.findSystemLocale();
 
-    final Completer<int> runCompleter = new Completer<int>();
-    Chain.capture<Future<Null>>(() async {
+    try {
```

Comment:
Actually, n/m -- he didn't add the `Chain.capture()`, and this is still semantically the same